test/db/cmd/cmd_pc

```diff
@@ -259,3 +259,67 @@ const uint8_t buffer[_BUFFER_SIZE] = {
 };
 EOF
 RUN
+
+NAME=pci1
+FILE=-
+CMDS=<<EOF
+e asm.arch=x86
+e asm.bits=32
+wa nop
+pci 1
+EOF
+EXPECT=<<EOF
+#define _BUFFER_SIZE 1
+const uint8_t buffer[_BUFFER_SIZE] = {
+ 0x90,                                 /* nop */
+};
+EOF
+RUN
+
+NAME=pci
+FILE=bins/elf/analysis/hello-linux-x86_64
+CMDS=<<EOF
+pci 29
+pci -10
```

Comment:
then someone has to explain me why `pci -10` should be converted to `pci 10` :D but that's a question for another day i guess.